lib/matplotlib/text.py

```diff
@@ -1553,9 +1553,8 @@ def _get_xy_transform(self, renderer, coords):
             return self.axes.transData
         elif coords == 'polar':
             from matplotlib.projections import PolarAxes
-            tr = PolarAxes.PolarTransform(apply_theta_transforms=False)
-            trans = tr + self.axes.transData
-            return trans
+            tr = PolarAxes.PolarTransform()
+            return PolarAxes.PolarTransform() + self.axes.transData
```

Comment:
Missed deleting `tr`:
```suggestion
            return PolarAxes.PolarTransform() + self.axes.transData
```